enderio-base/src/main/java/com/enderio/base/data/recipe/standard/MaterialRecipes.java

```diff
@@ -29,5 +41,231 @@ protected void buildCraftingRecipes(@Nonnull Consumer<FinishedRecipe> recipeCons
             .define('G', EIOItems.GRAINS_OF_INFINITY::get)
             .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(EIOItems.GRAINS_OF_INFINITY::get))
             .save(recipeConsumer);
+        
+        ShapedRecipeBuilder.shaped(Items.CAKE)
+            .pattern("MMM")
+            .pattern("SCS")
+            .define('M', Items.MILK_BUCKET)
+            .define('S', Items.SUGAR)
+            .define('C', EIOItems.CAKE_BASE.get())
+            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(EIOItems.CAKE_BASE.get()))
+            .save(recipeConsumer);
+        
+        ShapelessRecipeBuilder.shapeless(EIOItems.PHOTOVOLTAIC_COMPOSITE.get())
+            .requires(EIOTags.Items.DUSTS_LAPIS)
+            .requires(EIOTags.Items.DUSTS_COAL)
+            .requires(EIOTags.Items.SILICON)
+            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(EIOItems.SILICON.get()))
+            .save(recipeConsumer);
+        
+        ShapedRecipeBuilder.shaped(EIOItems.EMPTY_SOUL_VIAL.get())
+            .pattern(" S ")
+            .pattern("Q Q")
+            .pattern(" Q ")
+            .define('S', EIOItems.SOULARIUM_INGOT.get())
+            .define('Q', EIOTags.Items.FUSED_QUARTZ)
+            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(EIOItems.SOULARIUM_INGOT.get()))
+            .save(recipeConsumer);
+        
+        ShapedRecipeBuilder.shaped(EIOItems.UNFIRED_DEATH_URN.get())
+            .pattern("CPC")
+            .pattern("C C")
+            .pattern("CCC")
+            .define('C', Items.CLAY_BALL)
+            .define('P', EIOItems.PULSATING_POWDER.get())
+            .unlockedBy("has_ingredient", InventoryChangeTrigger.TriggerInstance.hasItems(EIOItems.PULSATING_POWDER.get()))
+            .save(recipeConsumer);
+        
+        blockToIngots(recipeConsumer, EIOItems.ELECTRICAL_STEEL_INGOT.get(),EIOBlocks.ELECTRICAL_STEEL_BLOCK.get());
```

Comment:
Doesn't registrate have a storageBlock builder? could we maybe try getting registrate over here to do that?
That may involve a bit of a recipe refactor though :/ Could become a todo for later if thats too much for this PR (as we'd need to extend registrate which as I learned last time is a little messier in a project this size than I thought it would be).